monkestation/code/modules/uplink/uplink_items/misc.dm

```diff
@@ -6,3 +6,20 @@
 	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
 	item = /obj/item/syndie_glue
 	cost = 2
+
+/datum/uplink_item/device_tools/neutered_borer_egg
+	name = "Neutered borer egg"
+	desc = "A borer egg specifically bred to aid operatives. \
+			It will obey every command and protect whatever operative they first see when hatched. \
+			Unfortunately due to extreme radiation exposure, they cannot reproduce."
+	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
+	item = /obj/effect/mob_spawn/ghost_role/borer_egg/neutered
+	cost = 20
+
+/datum/uplink_item/device_tools/borer_egg
```

Comment:
Im still HIGHLY iffy on this.

Reply:
same, but dunno where id fit em in otherwise

Reply:
putting them into mid-rounds. Resolves this